File: ExpenseSplitter.Backend/ExpenseSplitter.Api/Data/ExpensePartParticipant.cs
using System.ComponentModel.DataAnnotations;

namespace ExpenseSplitter.Api.Data
{
    public class ExpensePartParticipant
    {
        [Key] public int Id { get; set; }

        public int ExpensePartId { get; set; }
        public ExpensePart ExpensePart { get; set; }
        public int ParticipantId { get; set; }
        public Participant Participant { get; set; }
    }
}

File: ExpenseSplitter.Frontend/ExpenseSplitter.Web/src/app/data/expense-part-participant.ts
import { Participant } from './participant';
import { ExpensePart } from './expense-part';

export class ExpensePartParticipant {
    id: number;

    expensePartId: number;
    expensePart: ExpensePart;

    participantId: number;
    participant: Participant;
}
